libexec/scoop-info.ps1

```diff
@@ -76,66 +74,63 @@ if ($manifest.license) {
     } else {
         $license = "$($manifest.license) (https://spdx.org/licenses/$($manifest.license).html)"
     }
-    Write-Output "License: $license"
+    $item.License = $license
 }
 
 # Manifest file
-Write-Output "Manifest:`n  $manifest_file"
+$item.Manifest = $manifest_file
 
-if($status.installed) {
+if ($status.installed) {
     # Show installed versions
-    Write-Output "Installed:"
-    $versions = Get-InstalledVersion -AppName $app -Global:$global
-    $versions | ForEach-Object {
-        $dir = versiondir $app $_ $global
-        if($global) { $dir += " *global*" }
-        Write-Output "  $dir"
+    $installed_output = ""
+    Get-InstalledVersion -AppName $app -Global:$global | ForEach-Object {
+        $installed_output += "`n$(versiondir $app $_ $global)"
     }
-} else {
-    Write-Output "Installed: No"
+    $item.Installed = $installed_output
```

Comment:
Why add so many blank lines?

Reply:
This should be better:

![image](https://user-images.githubusercontent.com/5832170/154299655-8f2077cc-10e7-4292-b7be-1e595f5c9f03.png)

Reply:
It will become more clear when you have more than one versions installed. I was trying to be similar to how it was before (which had a newline in beginning).

Reply:
Since you use List View now, the output with blank line looks like a misformatted table. For old string-like output, indentation is suitable to clarify long sentence, but for the list (you've used a list with wider first column), line width is not worth considering.

Reply:
Okay, will change it. But we need to note that if I remove the blank line in beginning, it will be a breaking change for users who relied on non-PowerShell tools (such as `grep` and `findstr`) to gather information from the scoop-info command.

Reply:
This command is used mostly in console or single line command IMO, and is a easy-to-found format change, not breaking. We could change it, the sooner, the better.